File: src/test/java/no/nav/fo/veilarbregistrering/orgenhet/adapter/Norg2GatewayConfig.java
package no.nav.fo.veilarbregistrering.orgenhet.adapter;

import no.nav.fo.veilarbregistrering.orgenhet.Norg2Gateway;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;

import static org.mockito.Mockito.mock;

@Configuration
public class Norg2GatewayConfig {

    @Bean
    public Norg2Gateway norgGateway() {
        return mock(Norg2Gateway.class);
    }
}

File: src/test/java/no/nav/fo/veilarbregistrering/orgenhet/adapter/Norg2GatewayConfig.kt
package no.nav.fo.veilarbregistrering.orgenhet.adapter

import io.mockk.mockk
import no.nav.fo.veilarbregistrering.orgenhet.Norg2Gateway
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration

@Configuration
class Norg2GatewayConfig {

    @Bean
    fun norgGateway(): Norg2Gateway = mockk()
}
